Fits/T1 EWAR Cruisers/[NVY-30] Arbitrator.md

```diff
@@ -0,0 +1,52 @@
+# Arbitrator
+
+Entry EWAR Cruiser
+
+## Description
+
+Utilizes Tracking/Guidance Disruption to disrupt hostile DPS or Logistics. This ship is entirely reliant on range to avoid incoming DPS so do not burn close unless absolutely necessary.
+
+Tracking Disruptor - Use against ships with Guns/Turrets.
+Guidance Disruptor - Use against ships with Missiles.
+Range Disruption - Use against ships with long range capabilities.
+Tracking Speed / Precision Disruption - Use against bigger ships shooting smaller ships. 
+
+## Fit
+
+```
+[Arbitrator, [NVY-30] Arbitrator]
```

Comment:
Better as a brick IMO, can actually stay alive
```
[Arbitrator, [NVY-30] Arbitrator]

Damage Control II
1600mm Crystalline Carbonide Restrained Plates
Multispectrum Energized Membrane II
Multispectrum Energized Membrane II
Reactive Armor Hardener

50MN Quad LiF Restrained Microwarpdrive
DDO Scoped Tracking Disruptor I
DDO Scoped Tracking Disruptor I
DDO Scoped Tracking Disruptor I

Drone Link Augmentor I
Drone Link Augmentor I
Auto Targeting System I
[Empty High slot]

Medium Trimark Armor Pump I
Medium Trimark Armor Pump I
Medium Trimark Armor Pump I


Acolyte I x7
Hornet EC-300 x7
Vespa EC-600 x8


Missile Precision Disruption Script x3
Missile Range Disruption Script x3
Nanite Repair Paste x10
Optimal Range Disruption Script x3
Tracking Speed Disruption Script x3
Mobile Depot x1
Highstroke Scoped Guidance Disruptor x3
```

Reply:
The primary issue is that yeah you gain a a bit of raw EHP but you're also forcing yourself to actually be in range to take damage. That cuts the effective range of the fit from 135k down to 87.5k and that range is always going to be a better tank than a T1 cruiser's EHP.